### ISSUE TITLE
Port "Add Configuration..." to tasks

### ISSUE THREAD

**Issue by Beetix** [eclipse-theia/theia#7164]
### Description
Feature request as discussed with @akosyakov.

Port the behaviour of `Add Configuration...` to tasks. This means adding the possibility to add predefined tasks in an existing `tasks.json`. This feature also does not seem to be implemented in VS code.
Currently there is only the possibility to copy a whole template when using `Tasks: Configured tasks...` and `tasks.json` does not exist.

**Comment by akosyakov** [comment]
@RomanNikitenko @elaihau Maybe whether there is already something similar in Theia?

**Comment by vince-fugnitto** [comment]
> @RomanNikitenko @elaihau Maybe whether there is already something similar in Theia?

We prompt users to choose a pre-defined task if:
- a `tasks.json` does not exist
- a `tasks.json` exists but is empty

The proposal is to add a new menu entry/command which can be used to add **pre-defined** tasks in an **existing** `tasks.json` from what I understand. @Beetix is it something you'd want to look into?

**Comment by Beetix** [comment]
@vince-fugnitto Yes, this is the idea. I can look into it (I don't guarantee how fast 😅).

**Comment by vince-fugnitto** [comment]
> @vince-fugnitto Yes, this is the idea. I can look into it (I don't guarantee how fast 😅).

No problem :) It is definitely an enhancement but not critical so take the time you need!
